src/test/scala/cognite/spark/v1/RawTableRelationTest.scala

```diff
@@ -282,6 +285,33 @@ class RawTableRelationTest
     assert(df.count() == 10)
   }
 
+  it should "check partition sizes for partitions=10" taggedAs (ReadTest) in {
```

Comment:
We will have a for loop covering partitions=10 and partitions>10 cases when api is fixed.